jupyter-book/introduction/interoperability.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<p>I'd show how to move an anndata object to an SCE object and back in this section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/single-cell-best-practices/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>That's kinda shown in the next example but maybe we should be more explicit about it?</p>